l1-contracts/src/core/RollupCore.sol

```diff
@@ -424,18 +425,7 @@ contract RollupCore is
           interim.totalBurn += interim.burn;
 
           // Compute the proving fee in the fee asset
-          {
-            // @todo likely better for us to store this if we can pack it better
-            interim.feeAssetPrice = IntRollupLib.getFeeAssetPerEth(
```

Comment:
interim.feeAssetPrice can be removed from the interim variables struct

Reply:
The interim is getting rekt in the pr staked on top of this one. With a bigish move around as well